static/css/components/search-result-item.less

```diff
@@ -97,7 +105,28 @@
 }
 
 @media only screen and (min-width: @width-breakpoint-tablet) {
-  .searchResultItem {
-    flex-wrap: nowrap;
+  .list-books {
```

Comment:
Am not sure about this selector. 
It looks like you sre trying to increase specificity but I'd rather not do that if possible.

What happens when this is removed?

Reply:
Used selectors `list-books` and `list-results` to ensure the width of `details` remains different for pages lists and want-to-read